### PR TITLE
Added some qoutes.

### PR DESCRIPTION
**Pull request by Utkarsh299-tech**
Added some quotes in the input.py file.

### CODE REVIEW

File: bot/cogs/inputs.py [hunk 1]
```diff
@@ -1496,6 +1496,10 @@
     "The successful warrior is the average man, with laser-like focus.": "Bruce Lee",
     "Success? I don’t know what that word means. I’m happy. But success, that goes back to what in somebody’s eyes success means. For me, success is inner peace. That’s a good day for me.": "Denzel Washington",
     "A kiss is a lovely trick designed by nature to stop speech when words become superfluous.": "Ingrid Bergman",
+    "It is not the strongest or most intelligent who will survive but those who can best manage change.": "Charles Darwin",
```
Comment (Vyvy-vi) [inline bot/cogs/inputs.py:1499]: LGTM-!

**Review by Vyvy-vi — APPROVED** [review]

**Comment by Vyvy-vi** [comment]
Thank you for contributing to [TearDrops](https://github.com/Vyvy-vi/TearDrops/)-! 🎉

**Comment by Vyvy-vi** [comment]
@all-contributors add @Utkarsh299-tech for ideas

**Comment by Utkarsh299-tech** [comment]
Thanks for merging this PR! :)

**Pull request merged**